app/src/test/java/com/android/unio/model/event/EventViewModelTest.kt

```diff
@@ -95,3 +97,20 @@ class EventViewModelTest {
 
   class UnsupportedViewModel : ViewModel()
 }
+
+class MockReferenceList<T>(elements: List<T> = emptyList()) : ReferenceList<T> {
```

Comment:
Consider injecting functionality into the existing repository that you mock instead of making a mock repository every time. It is harder to figure out but ultimately less work in the long run. I can however also agree that making a mock repository can be easier to figure out

Reply:
Sure ! Done in #123 ! : )